2回目のプルリク

tsujike opened the pull request:
アウトラインに肉付けしてみました。ボリュームは詰め詰めなので、ここから不要なところを削ってブラッシュアップしていきます。

tsujike committed:
2回目のプルリク

アウトラインに肉付けしてみました。ボリュームは詰め詰めなので、ここから不要なところを削ってブラッシュアップしていきます。

ntakahashi0505 commented on [企画書]ノンプロ研定例会Vol51について.md:
```diff
@@ -127,36 +127,98 @@
+### ノンプロ研的ペアプロとは　6分
+#### リモート・ペア・プログラミング　2分
+#### レベル差のありなし　2分
+#### ノンプログラマーがペアプロをやるメリット・効果　2分
```
メリットは4章の「ペアプロの母」にまとめたほうがいいのかな？

tsujike replied:
対応しました！！

ntakahashi0505 commented on [企画書]ノンプロ研定例会Vol51について.md:
```diff
@@ -127,36 +127,98 @@
+#### ノンプログラマーがペアプロをやるメリット・効果　2分
+### 1章まとめ　1分
+
+## ２．ノンプロ研ペアプロ部（20：30～20：45）15／15分
```
つじけさんの自分語りも多く含まれるので、タイトル変えたほうがいいかな？

tsujike replied:
タイトル変更はおまちください。

ntakahashi0505 commented on [企画書]ノンプロ研定例会Vol51について.md:
```diff
@@ -127,36 +127,98 @@
+#### 自己関与力　1分
+#### アウトプットしよう　1分
+#### 卒業LTがゴールじゃない　1分
+### 心理的安全性　7分
```
中身わからないので考えすぎだったらごめんなさいなのですが、心理的安全性の話が複雑過ぎて、それまでの話がすっ飛ばないかやや心配です。

確かに、心理的障壁をどう超えるかという話は大事なのですが、心の問題を理論的に説得しにかかるのは逆効果のようにも思えます。

たとえば、つじけさん自身が手が震えるくらいドキドキしていたけどやってみたこうだった、とか、他のメンバーのおすすめのペアプロやその圧倒的な効果を事例として紹介するほうが、効果的かも知れません。

tsujike replied:
心理的安全性の話をボリュームダウンしました。
「先輩たちの声」コーナーを設けました。

ntakahashi0505 reviewed (changes requested):
気になる点をコメントしてみました。

構成についてはスライド作りながら変わってくると思います。
現時点でもだいぶクリアしていると思いますが、各パートの凝集度の高さをキープしていただければOKと思います。

よろしくお願いいたします～

tsujike committed:
3回目のプルリクです

1章のメリットを4章にまとめました。4章の心理的なやつをボリュームダウンして、「先輩方の声」という節を設けました。タイトル変更はお待ちください。「プログラミングスキルが爆上がり」というのがメインテーマですが、裏テーマとして「ペアプロを経験して変わったこと」みたいなのを伝えられたらいいなと思ってます。

tsujike commented:
あー、今朝の作業はプルリクじゃなくて、PUSHをこちらから再度行ったんですね。なるほど。プルしてくださいというのは、マージだったり、編集者が変更を加えたので、そちらも最新版にしてください（プルをリクエストする）って意味ですね。

ntakahashi0505 commented:
ありがとうございます。

もうマージしてもいいのですが、せっかくなのでこちらの見出しタイトルの対応をお待ちしてみたいと思います。
https://github.com/tsujike/teireikai211021/pull/2#discussion_r721148191

どうぞよろしくお願いいたします！

tsujike committed:
スライドを追加します

タイトルを決めるための、2章の自分語りがどんなものか、スライド起こしてみました。

tsujike commented:
アウトラインとしては、

1章　プロの（正式な）ペアプロってなに？
2章　つじけさんなんでペアプロをはじめたの？
3章　ノンプロ研ペアプロ広場の紹介
4章　みんなもペアプロやりましょう！

という内容です。

以下、共通認識です。
「ペアプロでプログラミングスキルが爆上がり！」
「みんなもペアプロやりましょう！」
「今回の定例会のLTだけで、仕組み化できるとは思っていない」
「まずは成功事例を拡散することを続ける」

これに、わたしにしか話せない内容を足すとするなら、
「なぜペアプロをしようと思ったのか」
「つじけさんも初心者で弱虫だった」
「ノンプロ研ペアプロのプロとしてのオススメ」
ということだと思います。
ただし、自分語りが強くならないように「みなさんもこういう状況になること、なったことありますよね」を意識していきたいですね。（相談する相手がいないとか、ついつい自分でやった方が早いと思ってしまうこととか）

タイトルですが、先頭にセミナー参加者のメリットを伝えるのは必要なので、
A『あなたのスキルが爆上がりする！』
はマストでいいと思います。

本職のペアプロではないので。「ノンプログラマー」って単語は入れたいですね。
A ＋ 『ノンプログラマーのためのペアプロ』
A ＋ 『ノンプログラマーによるペアプロ』
A ＋ 『ノンプログラマーのペアプロ術』
A ＋ 『ペアプロによるノンプログラマーのお悩み解決』

最後のやつ良さそうですね。ご検討お願いします。

ntakahashi0505 commented:
タイトル、最後のやつでOKっす。

ntakahashi0505 reviewed (approved):
マージしますね～
